modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -2265,8 +2266,106 @@ var UIDynamicList = UIElement.extend(/** @lends LuCI.ui.DynamicList.prototype */
 			this.addItem(dl, this.values[i], label);
 		}
 
+		this.initDragAndDrop(dl);
+
 		return this.bind(dl);
 	},
+	initDragAndDrop: function(dl) {
+		let draggedItem = null;
+		let touchStartY = 0;
+		let placeholder = null;
+
+		dl.addEventListener('dragstart', (e) => {
+			if (e.target.classList.contains('item')) {
+				draggedItem = e.target;
+				e.target.style.opacity = 0.5;
+			}
+		});
+
+		dl.addEventListener('dragend', (e) => {
+			if (e.target.classList.contains('item')) {
+				e.target.style.opacity = "";
+			}
+		});
```

Comment:
maybe 
```js
	dl.addEventListener('dragend', (e) => resetStyles(e.target));
```

Reply:
That may be better, but these events may get triggered by sub-items as well, so actually not sure.

Reply:
I tested. Worked fine.

Reply:
I know it works. Its just that maybe it get applied to something you dont want, e.g. in (future) extensions . Let me just remove it though.